### PR TITLE
Add daily test parsing script

### PR DESCRIPTION
**Pull request by romunov**
This script parses daily laboratory test data. Output contains the columns

```
        "datum",
        "tests.performed",
        "tests.performed.todate",
        "tests.positive",
        "tests.positive.todate",
        "tests.regular.performed",
        "tests.regular.performed.todate",
        "tests.regular.positive",
        "tests.regular.positive.todate",
        "tests.ns-apr20.performed",
        "tests.ns-apr20.performed.todate",
        "tests.ns-apr20.positive",
        "tests.ns-apr20.positive.todate",
        "tests.lab.imi.performed",
        "tests.lab.imi.performed.todate",
        "tests.lab.imi.positive",
        "tests.lab.imi.positive.todate",
        "tests.lab.nlzohmb.performed",
        "tests.lab.nlzohmb.performed.todate",
        "tests.lab.nlzohmb.positive",
        "tests.lab.nlzohmb.positive.todate",
        "tests.lab.nlzohlj.performed",
        "tests.lab.nlzohlj.performed.todate",
        "tests.lab.nlzohlj.positive",
        "tests.lab.nlzohlj.positive.todate",
        "tests.lab.golnik.performed",
        "tests.lab.golnik.performed.todate",
        "tests.lab.golnik.positive",
        "tests.lab.golnik.positive.todate",
        "tests.lab.nlzohkp.performed",
        "tests.lab.nlzohkp.performed.todate",
        "tests.lab.nlzohkp.positive",
        "tests.lab.nlzohkp.positive.todate",
        "tests.lab.nlzohnm.performed",
        "tests.lab.nlzohnm.performed.todate",
        "tests.lab.nlzohnm.positive",
        "tests.lab.nlzohnm.positive.todate",
        "tests.lab.sbce.performed",
        "tests.lab.sbce.performed.todate",
        "tests.lab.sbce.positive",
        "tests.lab.sbce.positive.todate",
        "tests.lab.nlzohkr.performed",
        "tests.lab.nlzohkr.performed.todate",
        "tests.lab.nlzohkr.positive",
        "tests.lab.nlzohkr.positive.todate",
        "tests.lab.sbng.performed",
        "tests.lab.sbng.performed.todate",
        "tests.lab.sbng.positive",
        "tests.lab.sbng.positive.todate",
        "tests.lab.sbsg.performed",
        "tests.lab.sbsg.performed.todate",
        "tests.lab.sbsg.positive",
        "tests.lab.sbsg.positive.todate",
        "tests.lab.nlzohms.performed",
        "tests.lab.nlzohms.performed.todate",
        "tests.lab.nlzohms.positive",
        "tests.lab.nlzohms.positive.todate",
```
where

```
tests.performed = tests.regular.performed + tests.ns-apr20.performed
tests.positive = tests.regular.positive + tests.ns-apr20.positive
```

### CODE REVIEW

**Review by jurijbajzelj — APPROVED** [review]

File: lab_tests/parse_daily_tests.py [hunk 1]
```diff
@@ -3,239 +3,258 @@
...
+            "tests.lab.sbsg.positive",
+            "tests.lab.sbsg.positive.todate",
+            "tests.lab.nlzohms.performed",
+            "tests.lab.nlzohms.performed.todate",
+            "tests.lab.nlzohms.positive",
+            "tests.lab.nlzohms.positive.todate",
+        ]
+    ]
+
+    xy.to_csv(path_or_buf=OUTPUT / "tests.csv", sep=",", index=False)
```
Comment (majazaloznik) [inline lab_tests/parse_daily_tests.py:325]: change to "lab-tests.csv"

File: lab_tests/parse_daily_tests.py [hunk 2]
```diff
@@ -3,239 +3,258 @@
...
+
+    timestamp = OUTPUT / "tests.csv"
+    update_time = int(time.time())
+
+    with open(f"{timestamp}.timestamp", "w") as f:
+        f.write(str(update_time))
+
+    return xy
```
Comment (majazaloznik) [inline lab_tests/parse_daily_tests.py:333]: delete this line?

File: lab_tests/parse_daily_tests.py [hunk 3]
```diff
@@ -3,239 +3,258 @@
...
+            try:
+                end_datum = interval.split("-")[1]
+                end_datum = datetime.datetime.strptime(end_datum, "%d.%m.%Y")
+            except IndexError as ie:
+                pass
+            xy.loc[index, "datum"] = end_datum
+
+    # The table comes with dates in the future. Since some cells may be polluted,
+    # the easiest thing to do is to remove them.
+    xy = xy[xy["datum"] <= datetime.datetime.now()]
```
Comment (majazaloznik) [inline lab_tests/parse_daily_tests.py:98]: this should be `now - 1`

but really should be the parameter that is used to find the filename

File: lab_tests/parse_daily_tests.py [hunk 4]
```diff
@@ -3,239 +3,258 @@
...
-
-xy["tests.lab.nlzohms.performed.todate"] = xy["tests.lab.nlzohms.performed"].fillna(value=0)
-xy["tests.lab.nlzohms.performed.todate"] = xy["tests.lab.nlzohms.performed.todate"].cumsum()
-xy["tests.lab.nlzohms.positive.todate"] = xy["tests.lab.nlzohms.positive"].fillna(value=0)
-xy["tests.lab.nlzohms.positive.todate"] = xy["tests.lab.nlzohms.positive.todate"].cumsum()
-
-xy = xy[
-    [
+
+def parse_daily_tests(xlsx=None):
```
Comment (majazaloznik) [inline lab_tests/parse_daily_tests.py:13]: set default xlsx param to 
`"/data/Dnevno laboratorijsko poročilo o testiranju na okužbo s SARS-CoV-2 za <now-1>.xlsx"`
where <now-1> is yesterday's date

Comment (romunov) [reply lab_tests/parse_daily_tests.py:13]: I'm thinking more in terms of just giving it a folder and it finds the latest file. If you want a specific date, you can pass it an xlsx parameter.

**Review by majazaloznik — CHANGES_REQUESTED** [review]
change the default path in the README file

File: lab_tests/parse_daily_tests.py (hunk 4 above)
Comment (majazaloznik) [reply lab_tests/parse_daily_tests.py]: fine by me

File: lab_tests/README.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ Other actions:
 To run the script using a specific file, do
 
 ```bash
-python parse_daily_tests.py --xlsx ./path/to/file.xlsx
+python parse_daily_tests.py --xlsx ./data/path/to/file.xlsx
```
Comment (majazaloznik) [inline lab_tests/README.md:45]: this is an example path, why add `/data` here? revert this change please and instead change the default in line 28 to  `/data`

**Review by majazaloznik — APPROVED** [review]
